skbio/io/format/genbank.py

```diff
@@ -0,0 +1,669 @@
+
+    return res, pd.Series(pmd)
+
+
+def _parse_origin(lines):
+    '''Parse the ORIGIN section for sequence.
+    '''
+    sequence = []
+    for line in lines:
+        if line.startswith('ORIGIN'):
+            continue
+        # remove the number at the beg of each line
+        items = line.split()
+        sequence.append(''.join(items[1:]))
+    return ''.join(sequence)
+
+
+def _serialize_origin(seq, indent=9):
+    '''Serialize seq to ORIGIN.
+
+    Parameters
+    ----------
+    seq : str
+    '''
+    n = 1
+    line_size = 60
+    frag_size = 10
+    for i in range(0, len(seq), line_size):
+        line = seq[i:i+line_size]
+        s = '{n:>{indent}} {s}\n'.format(
+            n=n, indent=indent, s=chunk_str(line, frag_size, ' '))
+        if n == 1:
+            s = 'ORIGIN\n' + s
+        n = n + line_size
+        yield s
+
+
+def _parse_section_default(
+        lines, label_delimitor=None, join_delimitor=' ', return_label=False):
+    '''Parse sections in default way.
+
+    Do 2 things:
+        1. split first line with label_delimitor for label
+        2. join all the lines into one str with join_delimitor.
+    '''
+    data = []
+    first = True
+    label = None
+    for line in lines:
+        if first:
+            try:
+                items = line.split(label_delimitor, 1)
+            except:
+                GenbankFormatError('Could not split the line:\n%s', line)
+            if len(items) == 2:
+                label, section = items
+            else:
+                label = items[0]
+                section = ""
+            data.append(section)
+            first = False
+        else:
+            data.append(line)
+    data = join_delimitor.join(i.strip() for i in data)
+    if return_label:
+        return label, data
+    else:
+        return data
+
+
+def _serialize_section_default(header, obj, indent=12):
+    return '{header:<{indent}}{obj}\n'.format(
+        header=header, obj=obj, indent=indent)
+
+
+def yield_section(is_another_section, **kwargs):
+    '''Returns function that returns successive sections from file.
+
+    Parameters
+    ----------
+    is_another_section : callable
+        It takes a string as input and return a boolean indicating
+        a new section starts.
+    kwargs : dict, optional
+        Keyword arguments will be passed to `_line_generator`.
+
+    Returns
+    -------
+    function
+        A function accept a list of lines as input and return
+        a generator to yield section one by one.
+    '''
+    def parser(lines):
+        curr = []
+        for line in _line_generator(lines, **kwargs):
+            # if we find another, return the previous section
+            if is_another_section(line):
+                if curr:
+                    yield curr
+                    curr = []
+            curr.append(line)
+        # don't forget to return the last section in the file
+        if curr:
+            yield curr
+    return parser
+
+
+_PARSER_TABLE = {
```

Comment:
can I? The functions appears after that dict definition. I got flake8 error at least.